internal/git/tag_test.go

```diff
@@ -199,7 +242,7 @@ func gitTagCommit(t *testing.T, path, tag string) string {
 	}
 
 	// why the ~1? we want git log to return inclusive results
```

Comment:
> `// why the ~1`

`~1` doesn't appear in the line below. I would, however, love to know why `"-1` is in the the next line.

Reply:
negative 1 vs approximately 1! jk, I'll update the comment to reflect the command (I think I'll do that for a few other git command helpers too). In short the `-1` will only show the first result for `git log` instead of traversing further through commits.